docs/tempo/website/configuration/override.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Override trace limits
+weight: 30
+---
+
+# Override trace limits
+
+The default per user or per tenant trace limits in Tempo may not be sufficient in high volume tracing environments. For example. the following message indicates that your tenants are running up against the trace limits imposed on them.
+ ```
+    max live traces per tenant exceeded: per-user traces limit (local: 10000 global: 0 actual local: 10000) exceeded
+```    
+
+In such environments, bump up the limits either globally across the cluster or individually by tenants
+so that your traces are not limited by the default settings. 
+
+This topic lists the parameters whose limits can be overridden, as well as describes various methods of
+overriding these default limits.
+
+## Override parameters
+
+The parameters whose default values can be overridden are:
+
+   - `ingestion_burst_size` : `NEED INFO`
+   - `ingestion_max_batch_size` : Per-user allowed ingestion max batch size (in number of spans). Default is `1000`.
+   - `ingestion_rate_limit` : Per-user ingestion rate limit in spans per second. Default is `100,000`.
+   - `max_spans_per_trace` : Maximum number of spans per trace.  0 to disable. Default is `50,000`.
+   - `max_traces_per_user`: Maximum number of active traces per user, per ingester. 0 to disable. Default is `10,000`.
+
+`NEED INFO:The trigger points for each of the trace limits, and the error message to help the user identify which trace limit has been hit.`
```

Comment:
`ingestion_burst_size` and `ingestion_rate_limit` both control the rate limit.  A log message would contain:
```
RATE_LIMITED: ingestion rate limit (100000 spans) exceeded while adding 10 spans
```
`max_spans_per_trace`:
```
TRACE_TOO_LARGE: totalSpans (50000) exceeded while adding 2 spans
```
`max_traces_per_user`:
```
LIVE_TRACES_EXCEEDED: max live traces per tenant exceeded: per-user traces limit (local: 10000 global: 0 actual local: 1) exceeded
```

There is also a metric that tracks this information: `tempo_discarded_spans_total`.  It has a `reason` and a `tenant` label.  The `reason` label will contain either: `trace_too_large`, `live_traces_exceeded` or `rate_limited`.  `rate_limited` applies to both the rate limit and burst size.